data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/defense/defense_affect_everybody.cfg

```diff
@@ -38,14 +38,13 @@
                 apply_to = new_ability
                 [abilities]
                     {TEST_ABILITY attacks 100 ([affect_adjacent][/affect_adjacent]) ENEMIES=yes SELF=yes ALLIES=yes}
-                    {TEST_ABILITY defense 0 (apply_to=opponent
-                    [affect_adjacent][/affect_adjacent]) ENEMIES=yes SELF=yes ALLIES=yes}
+                    {TEST_ABILITY defense 0 ([affect_adjacent][/affect_adjacent]) ENEMIES=yes SELF=yes ALLIES=yes}
```

Comment:
Just noticed that the documentation on this test is reversed, at the top of the file it says that this is set to 100.

It's set to zero, and all attacks hit.